packages/swayze/lib/src/widgets/table_body/gestures/table_body_gesture_detector.dart

```diff
@@ -73,27 +89,23 @@ _TableGestureDetails _getTableGestureDetails(
 
   final box = context.findRenderObject()! as RenderBox;
   final localPosition = box.globalToLocal(globalPosition);
-  final positionResultX = viewportContext.pixelToPosition(
-    localPosition.dx,
-    Axis.horizontal,
-  );
 
-  final positionResultY = viewportContext.pixelToPosition(
-    localPosition.dy,
-    Axis.vertical,
-  );
+  final hoverResult = viewportContext.evaluateHover(localPosition);
```

Comment:
Is there any performance impact with this addition? Is this executed while we're scrolling? 


Excel seems to disable this type of checks when a user starts to scroll, and only enables it again when the mouse moves again:

https://user-images.githubusercontent.com/1267623/173531968-f97d51e0-6c31-4bc9-b361-b83b97e20d99.mov

Reply:
> "early optimization is the root of all evil"

I completely agree, that's why I asked if there's any performance impact. We currently don't actively run and monitor perf tests, so we need to run them manually and check if any regression was done. (You can manually run the perf tests in the main branch on `swayze_rows` and then compare with the branch that uses this version of Swayze).

Reply:
I need to go deeper into the performance tests -- and I have an idea to cache the hover so we don't have to re-calculate it every time, but I'm already marking this as ready to review, and will work on the performance in a new card that I'm adding.

Reply:
main branch: https://github.com/rows/native-apps/runs/6940534988
this branch: https://github.com/rows/native-apps/runs/6941950486

We seem to have gains on build times in certain resolutions, but we do lose some rasterizer times. It's so weird comparing the 2k vs 4k results from both branches.